lib/modules/manager/circleci/index.ts

```diff
@@ -1,8 +1,9 @@
 import { DockerDatasource } from '../../datasource/docker';
 import { OrbDatasource } from '../../datasource/orb';
 import { extractPackageFile } from './extract';
+import type { getRangeStrategy } from './range';
```

Comment:
```suggestion
export { getRangeStrategy } from './range';
```

type export would be removed after compile 😉